diagram.md

```diff
@@ -180,19 +180,19 @@ While EBML allows elements of the same level to be in no particular order, for b
 
 Here is a more complex representation of a Matroska file. This one lists some of the elements for examples. Each of these elements are described in the [specs]({{site.baseurl}}/index.html).
 
-The [Header]({{site.baseurl}}/index.html#EBMLHeader) must occur at the beginning of the file. This is how the library knows whether or not it can read the file. The design of EBML is pretty straight forward and is its own project on [SourceForge](http://ebml.sourceforge.net/). EBML is not just for Matroska, there are many different potential applications for it. As such, there is the possibility of there being new versions, such as a 2.0 design. The [EBMLVersion]({{site.baseurl}}/index.html#EBMLVersion) element would let the parser know first if it can read this file at all. If the EBMLVersion is set to 2.0, and the library is only able to read up to 1.2, then it knows it shouldn't even attempt to read this file. 
+The [Header]({{site.baseurl}}/index.html#EBMLHeader) MUST occur at the beginning of the file. This is how the library knows whether or not it can read the file. The design of EBML is pretty straight forward and is its own project on [SourceForge](http://ebml.sourceforge.net/). EBML is not just for Matroska, there are many different potential applications for it. As such, there is the possibility of there being new versions, such as a 2.0 design. The [EBMLVersion]({{site.baseurl}}/index.html#EBMLVersion) element would let the parser know first if it can read this file at all. If the EBMLVersion is set to 2.0, and the library is only able to read up to 1.2, then it knows it SHOULD NOT even attempt to read this file. 
```

Comment:
Because of our deadline, I suggest to address this more comprehensively later. The URLs don't render in the text based RFC anyhow.